src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -2,9 +2,19 @@
 
 import static org.testng.Assert.assertEquals;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import static org.testng.Assert.assertTrue;
```

Comment:
If you do `Ctrl-Shift-O` in Eclipse it will automatically re-order your imports. I mention it because that puts static imports together up at the top, and helps make the imports consistent across files